### ISSUE TITLE
Cannot view run query button on 13" screen with between keys

### ISSUE THREAD

**Issue by garrensmith** [apache/couchdb-fauxton#1221]
## Expected Behavior
I should be able to run a query

## Current Behavior
When using my 13" macbook, I cannot get to the run query button.

## Possible Solution
I think we can just scroll down.

## Steps to Reproduce (for bugs)

1. Make sure browser height is set to around 730px
2. Open query options and click on "between keys"
3. Try and run the query

<img width="1279" alt="Screenshot 2019-09-05 14 49 21" src="https://user-images.githubusercontent.com/179458/64343561-06cfef00-cfed-11e9-868c-f8f4187b499c.png">

## Your Environment

* Version used:
* Browser Name and version: Chrome latest
* Operating System and version (desktop or mobile): Mac Osx

**Comment by garrensmith** [comment]
As a quick test doing something like:

```
    #query-options-tray {
     overflow-y: scroll;
    max-height: 600px;
}
```

fixed it for me.

**Comment by popojargo** [comment]
Fixed with #1270
